docs/encoder.md

```diff
@@ -42,27 +48,67 @@ SDA = board.GP0
 SCL = board.GP1
 i2c = busio.I2C(SCL, SDA)
 
-encoder_handler.pins = ((i2c, 0x36, False),)
+encoder_handler.pins = ((i2c, 0x36, False), (encoder 2 definition), etc. )
 ```
 
+3. Define the mapping of keys to be called for each layer.
 
-3. Define the mapping of keys to be called (1 / layer)
 ```python
 # You can optionally predefine combo keys as for your layout
 Zoom_in = KC.LCTRL(KC.EQUAL)
 Zoom_out = KC.LCTRL(KC.MINUS)
 
-
-encoder_handler.map = [(( KC.VOLD, KC.VOLU, KC.MUTE),(encoder 2 definition), etc. ), # Layer 1
-                      ((KC.Zoom_out, KC.Zoom_in, KC.NO),(encoder 2 definition), etc. ), # Layer 2
-                      ((KC.A, KC.Z, KC.N1),(encoder 2 definition), etc. ), # Layer 3
-                      ((KC.NO, KC.NO, KC.NO),(encoder 2 definition), etc. ), # Layer 4
+encoder_handler.map = [ (( KC.VOLD, KC.VOLU, KC.MUTE), (encoder 2 definition), etc. ), # Layer 1
+                        ((Zoom_out, Zoom_in, KC.NO), (encoder 2 definition), etc. ), # Layer 2
+                        ((KC.A, KC.Z, KC.N1), (encoder 2 definition), etc. ), # Layer 3
+                        ((KC.NO, KC.NO, KC.NO), (encoder 2 definition), etc. ), # Layer 4
                       ]
 ```
 
+## Encoder resolution
+
+Depending on your encoder precision, it may send 4 or 2 pulses on every detent. By default the encoder resolution is set to 4, but if your an encoder only activates on every second detent (skips pulses), set the resolution to 2. If the encoder activates twice on every detent, set the value to 4.
```

Comment:
```suggestion
Depending on your encoder precision, it may send 4 or 2 pulses on every detent. By default the encoder resolution is set to 4, but if your encoder only activates on every second detent (skips pulses), set the resolution to 2. If the encoder activates twice on every detent, set the value to 4.
```